src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,14 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Each package in a Java project should include a package-info.java file.
+ * In that file declare package annotations.
+ * </p>
+ * <pre>
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation;
```

Comment:
This read code of this Check to get to know when it violate package statement and when not. We need two examples: with violation (with trailing comment "violation") and without violation ( training comment "ok").

This Check is a bit unusual, as it depends on file name. So you need two code blocks with text above on what is name of file.

Reply:
Added file names which doesn't violates and violates naming convention.

Reply:
We need file name and their content. Your examples should show how Check works, it is based on file name and its content.

Reply:
Changes done.Please review.